### ISSUE TITLE
"Kubernetes - Pods" Prometheus Dashboard Not Displaying Data

### ISSUE THREAD

**Issue by yongk802** [fabric8io/fabric8#6483]
When launching the "metrics" template, there is a "Kubernetes - Pods" dashboard that gets created to view the pod metrics. However, it no longer works. I believe the issue is with the queries in the templates. For example, I noticed one query was:

"query": "container_start_time_seconds{namespace=\"$namespace\"}"

but "namespace" actually needs to be "container_label_io_kubernetes_pod_namespace" for the query to return data from prometheus, so it should be:

"query": "container_start_time_seconds{container_label_io_kubernetes_pod_namespace=\"$namespace\"}"

I'm not sure how these dashboard templates are created though. Can someone update these templates or point me in the right direction to update them?

**Comment by yongk802** [comment]
I fixed the template so it correctly displays data. Not sure where this lives in the actual code, but this json can be used to update the template so it works. I renamed it "Kubernetes - Pods2" and have it here as a .txt instead of .json so it can be uploaded to github for anyone who wants to use it.

[kube-pod-metrics-dashboard.txt](https://github.com/fabric8io/fabric8/files/549037/kube-pod-metrics-dashboard.txt)
